servers.json

```diff
@@ -1338,5 +1338,23 @@
         "addresses": [
             "tearfulmc.com"
         ]
+    },
+        {
+        "id": "brawl",
+        "name": "Brawl",
+        "addresses": [
+            "mcbrawl.com",
+            "mc.brawl.com",
+            "brawl.com",
+            "brawl.network",
+            "mc-war.com",
+            "minecraftparty.com",
+            "mcctf.com",
+            "mcpvp.com",
+            "minecraftpvp.com",
+            "1.ctfmatch.brawl.com",
+            "2.ctfmatch.brawl.com",
+            "3.ctfmatch.brawl.com"
```

Comment:
Lunar Client automatically detect sub-domains. All of these are unnecessary. Please keep the following:
 - mcbrawl.com
 - brawl.com
 - brawl.network
 - mc-war.com
 - minecraftparty.com
 - mcctf.com
 - mcpvp.com
 - minecraftpvp.com

Reply:
oke